source/renderer/app/components/widgets/forms/WalletsDropdownOption.js

```diff
@@ -4,30 +4,61 @@ import classnames from 'classnames';
 import { getColorFromRange } from '../../../utils/colors';
 import styles from './WalletsDropdownOption.scss';
 import StakePool from '../../../domains/StakePool';
+import LoadingSpinner from '../LoadingSpinner';
 
 export type WalletOption = {
   delegatedStakePool?: ?StakePool,
-  detail: string,
+  detail?: string,
   label: string,
   numberOfStakePools?: number,
   selected?: boolean,
+  syncing?: boolean,
+  syncingLabel?: string,
 };
 
 export default class WalletsDropdownOption extends Component<WalletOption> {
   renderLabelAndTicker = () => {
```

Comment:
Please, check if it makes sense to create a different variable for the `isSyncing` scenario, so it doesn't need the ternary operator inside of the markup, which makes it harder to understand. E.G.:

```js
renderLabel = () => {...}
renderLabelSyncing = () => {...}
...
{
  isSyncing 
    ? this.renderLabel() 
    : this.renderLabelSyncing()
}
```

Reply:
Added